<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>发表论文</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>论文名称</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_paper_name">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>发表源</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>通讯作者</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_paper_source">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_contact_author">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_author">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>发表年份</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pub_year">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>类型</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>级别</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_paper_type">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_paper_cat">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_add_paper">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>添加论文</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="tree_widget_papers">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>681</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>论文名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>发表源</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>发表年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>论文类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>论文级别</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="button_search_paper">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>查询论文</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete_paper">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>删除论文</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>论文编号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_paper_number">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_clear_paper">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>承担项目</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>教师工号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prog_teacher_number">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>项目号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>承担经费</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>项目排名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prog_id">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prog_cur">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prog_rank">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_add_prog">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>添加项目</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete_prog">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>删除项目</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_search_prog">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>查找项目</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_refresh_prog">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="tree_widget_prog">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>751</width>
        <height>291</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>项目号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>项目名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>项目来源</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>项目类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>总经费</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开始年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>结束年份</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>主讲课程</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>课程号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>工号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>年份</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>学期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>承担学时</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_course_id">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_course_teacher_no">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_course_year">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_course_semester">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_course_time">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_add_course">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>添加课程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_delete_course">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>删除课程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_search_course">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>查询课程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_refresh_course">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="tree_widget_course">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>731</width>
        <height>271</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>课程号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学时数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程性质</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>统计查询</string>
     </attribute>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>教师工号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_stat_id">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>年份范围</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_stat_year">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_search_stat">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_stat_clear">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="tree_widget_stat_paper">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>721</width>
        <height>121</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>论文名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>发表年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>发表源</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>论文排名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>是否通讯作者</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="tree_widget_stat_prog">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>721</width>
        <height>141</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>项目名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>项目来源</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>承担经费</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>项目排名</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开始年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>结束年份</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="tree_widget_stat_course">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>360</y>
        <width>721</width>
        <height>141</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>课程号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>课程名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>年份</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>学期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>承担学时</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
